workflows/rnaseq/rnaseq-variant-calling.wdl

```diff
@@ -0,0 +1,132 @@
+# SPDX-License-Identifier: MIT
+# Copyright St. Jude Children's Research Hospital
+version 1.1
+
+import "../../tools/picard.wdl"
+import "../../tools/gatk4.wdl" as gatk
+
+workflow rnaseq_variant_calling {
+    meta {
+        description: "Call variants from RNA-Seq data. Produces a VCF file of variants. Based on GATK RNA-Seq short variant calling best practices pipeline."
+        outputs: {
+            recalibrated_bam: "BAM that has undergone recalibration of base quality scores"
+            recalibrated_bam_index: "Index file for recalibrated BAM file"
+            merged_vcf: "VCF of raw RNA-Seq variant calls"
+            merged_vcf_index: "Index for merged VCF file"
+            variant_filtered_vcf: "VCF file after variant filters have been applied"
+            variant_filtered_vcf_index: "Index for filtered variant VCF file"
+        }
+    }
+
+    parameter_meta {
+        bam: "BAM file of aligned RNA-Seq reads"
+        bam_index: "Index file for BAM file"
+        fasta: "Reference FASTA file"
+        fasta_index: "Index file for reference FASTA file"
+        dict: "Sequence dictionary for reference FASTA file"
+        callingIntervalList: "Interval list of regions from which to call variants. Used for parallelization."
+        knownVcfs: "Array of known indels VCF files"
+        knownVcfIndexes: "Array of index files for known indels VCF files"
+        dbSNP_vcf: "dbSNP VCF file"
+        dbSNP_vcf_index: "Index file for dbSNP VCF file"
+        scatterCount: "Number of intervals to scatter over"
```

Comment:
Should we have guidance on setting this? `6` seems like a magic number to me. (i.e. I do not understand why it was picked)

Reply:
It is definitely a magic number. I copied it from the GATK workflow. I can't find any real guidance for setting it. In fact in many places, the Broad pipelines leave it unset and completely up to the user. [This](https://gatk.broadinstitute.org/hc/en-us/community/posts/360058099652-Scatter-Count-recommendations) is about the only thing I can find and it isn't overly helpful. The [documentation](https://gatk.broadinstitute.org/hc/en-us/articles/360037592651-IntervalListTools-Picard#--SCATTER_COUNT) is similarly vague.

Reply:
I think it is also dependent on the value of `subdivision_mode`.

Reply:
Just bringing in 2 quotes from that forum answer:
> We usually use 10-20 for exomes. For our purposes this is a good compromise between wall clock time and the overhead of starting VMs and delocalizing. 

> If you're not using the cloud you could increase the scatter count, but there's no real need unless you're really in a hurry.

So the balance here is greater `scatter_count` leads to quicker turnaround (up to a point) but at higher costs for spinning up instances. That's fairly straightforward. Should we add a sentence saying something to the effect "This should typically be set to 5-20."?